rust-src/concordium_base/src/base.rs

```diff
@@ -0,0 +1,1239 @@
+//! Basis type definitions that are used throughout the crate.
```

Comment:
Well it was called `basic.rs` before. And the reaon is that these are just the "basic" types, and then there are other modules, such as updates and transactions which define further types.

So this is not exactly the same as the types.rs modules in smaller crates.

But of course it could be named that as well.